cmd/relayproxy/log/logger.go

```diff
@@ -1,10 +1,26 @@
 package log
 
 import (
+	"os"
+
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 )
 
-func InitLogger() *zap.Logger {
-	logger, _ := zap.NewProduction()
+type Logger struct {
+	ZapLogger *zap.Logger
+	Atom      zap.AtomicLevel
+}
+
+func InitLogger() *Logger {
+	logger := &Logger{Atom: zap.NewAtomicLevel()}
+
+	encoderCfg := zap.NewProductionEncoderConfig()
+	logger.ZapLogger = zap.New(zapcore.NewCore(
+		zapcore.NewJSONEncoder(encoderCfg),
+		zapcore.Lock(os.Stdout),
+		logger.Atom,
+	))
```

Comment:
I'll prefer to avoid having to set any configuration manually except the level and rely on the `NewProductionConfig()` helper.
```suggestion
        prodConfig := zap.NewProductionConfig()
        prodConfig.Level = logger.Atom
	logger.ZapLogger = prodConfig.Build()
```